cmd/up.go

```diff
@@ -170,7 +182,52 @@ List of components that kubeapps up installs:
 
 func init() {
 	RootCmd.AddCommand(upCmd)
-	upCmd.Flags().Bool("dry-run", false, "Provides output to be submitted to the server.")
+	upCmd.Flags().Bool("dry-run", false, "Show manifest to be submitted to the k8s cluster without deploying.")
+	upCmd.Flags().StringP("out", "o", "yaml", "Provides manifest format: yaml | json")
```

Comment:
I would add that this is only used for `--dry-run`

Reply:
In fact `-o` defaults to `yaml` so only `--dry-run` will also work. But yes, `-o` is used only for `--dry-run`

Reply:
what I mean is, this has no effect if I use `-o` without `--dry-run`, which might be confusing? So for that reason just changing the description to something like: "Dry-run manifest output format: yaml | kson" would clear that up IMO